_pages/en_US/uninstall-cfw.txt

```diff
@@ -26,7 +26,16 @@ Note that if you have any payload files other than `GodMode9.firm` in the `/luma
 
 ### Instructions
 
-#### Section I - Safety Test
+#### Section I - Checking for Flipnote
+1. Open system settings
+1. Click Internet
+1. Click DS Conncetion Settings
+1. If you see Flipnote, you'll need to copy the clean save, follow these steps: https://3ds.hacks.guide/installing-boot9strap-(fredtool)
+1. Now close system settings
+1. Now open DS Download Play
+1. If it shows Flipnote, open Homebrew Launcher, run frogtool and hit RESTORE patched DS Download play
```

Comment:
Assumes that people have frogtool *and* the ds download play backup.

Reply:
Ah, is DS DLP unrecoverable without it or are there steps you can take to retrieve it?

Reply:
The lazy way would be Twlfix which would hopefully cover both this and connection settings. Generally prefer to just get people with broken stuff to ask for help though so that we can catch their reasoning for uninstalling cfw